### ISSUE TITLE
Feature: export/import

### ISSUE THREAD

**Issue by jamonholmgren** [infinitered/thesis-phoenix#13]
### Export

In the settings dialog for any page, have a button to "Export Page Content" and "Export Site Content". These build a simple .json file, structured like this:

``` javascript
{
  meta: {
    generated_at_url: "http://example.com/contacts",
    generated_at: 2015-09-27T18:30:49-0300
  },
  pages: {
    "/": {
      title: "Home page",
      description: "something",
      page_contents: {
        "HEADER": { type: "text", contents: "Welcome to the Jungle!" }, 
        "SIDEBAR": { type: "html", contents: "<p>Whoop whoop</p>" }
      }
    },
    "/about": {
      // ...
    }
  }
}
```
### Import

In the settings dialog for any page, have a button to "Import Page Content" or "Import Site Content".

Questions:
- When doing Import Page Content, do we only fetch the page that matches the URL of the current page? If there's only one page in the json file, do we just import that one to the current page regardless of what the URL is?
- When importing the whole site, do we overwrite everything that already exists?

**Comment by yulolimum** [comment]
@jamonholmgren - I just noticed this issue. lol. We resolved a very similar issue with the last thesis version. 

I would like to discuss importing/exporting the entire site though. It is a good idea but I'm not sure that it is necessary for thesis for what it is. It is also very very destructive. Having said that, I think that transferring entire site's content should be done by a developer and manually. That's how we always did it with C5 and even Wordpress. Staging to production migrations were always straight up DB clones.

A few other thoughts:
- How to handle image transfers? If the site is using the built-in repo-uploader, that's no problem. If not, how to you differentiate thesis-uploaded images from any other images/files?  
- Thesis doesn't have a site-wide/global config/admin panel. Adding a site import to any of the current trays doesn't make sense.
